extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -53,3 +53,24 @@ amp-carousel .amp-carousel-button.amp-disabled {
   background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="#fff" viewBox="0 0 18 18"><path d="M9 3L7.94 4.06l4.19 4.19H3v1.5h9.13l-4.19 4.19L9 15l6-6z" /></svg>');
   background-size: 18px 18px;
 }
+
+.amp-carousel-button-start-hint .amp-carousel-button{
+  animation: -amp-carousel-hint 1s ease-in 0s 1 normal both;
+}
+
+@keyframes -amp-carousel-hint {
+  0%{
+    opacity: 1;
+    visibility: visible;
+  }
+  100% {
+    opacity: 0;
```

Comment:
So, my fear is that this will overwrite the values as set up by `amp-mouse-mode` and `[controls]` above. Can you test against those?